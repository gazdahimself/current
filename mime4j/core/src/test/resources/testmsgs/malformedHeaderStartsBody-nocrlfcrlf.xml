<message>
<header>
<field>
Subject: this is a subject</field>
</header>
<body>
This is an invalid header

Body text
</body>
</message>
