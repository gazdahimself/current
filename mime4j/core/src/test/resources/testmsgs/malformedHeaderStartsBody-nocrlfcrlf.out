Subject: this is a subject

This is an invalid header

Body text
